neuro_san/internals/chat/async_collating_queue.py

```diff
@@ -60,7 +60,7 @@ async def __anext__(self) -> Dict[str, Any]:
                 Will throw StopAsyncIteration when the final item is detected
                 via the is_final_item() method..
         """
-        message = await self.queue.get()
+        message = await self.queue.async_q.get()
         if self.is_final_item(message):
```

Comment:
This is how janus queue is special: for getting things _from_ a queue,
we use async get from async "side" of this queue.